### PR TITLE
[ADD] disable our cronjob for demo mode

### LINKED ISSUE #458: runbot 9.0 red due to letsencrypt?
Hi,

It seems the 9.0 branch is red on runbot due to the letsencrypt module?

```
Call of self.pool.get('letsencrypt').cron(cr, uid, *()) failed in Job 2
Traceback (most recent call last):
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3148182-9-0-209efa/openerp/addons/base/ir/ir_cron.py", line 129, in _callback
    getattr(model, method_name)(cr, uid, *args)
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3148182-9-0-209efa/openerp/api.py", line 250, in wrapper
    return old_api(self, *args, **kwargs)
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3148182-9-0-209efa/openerp/api.py", line 354, in old_api
    result = method(recs, *args, **kwargs)
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3148182-9-0-209efa/openerp/addons/letsencrypt/models/letsencrypt.py", line 151, in cron
    account_key, csr, acme_challenge, log=_logger, CA=DEFAULT_CA)
  File "/srv/openerp/instances/openerp-oca-runbot/sandbox/local/lib/python2.7/site-packages/acme_tiny.py", line 104, in get_crt
    raise ValueError("Error requesting challenges: {0} {1}".format(code, result))
ValueError: Error requesting challenges: 400 {
  "type": "urn:acme:error:malformed",
  "detail": "Error creating new authz :: Invalid character in DNS name",
  "status": 400
}
```

@hbrunn

### PR DESCRIPTION
**Pull request by hbrunn**
fixes #458

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

File: auth_supplier/security/auth_supplier_security.xml [hunk 1]
```diff
@@ -4,7 +4,6 @@
     <record id="group_auth_supplier" model="res.groups">
         <field name="name">Supplier Portal</field>
         <field name="category_id" ref="base.module_category_extra"/>
-        <field name="is_portal" eval="True"/>
```
Comment (pedrobaeza) [inline auth_supplier/security/auth_supplier_security.xml:7]: Why this change?

**Comment by elicoidal** [comment]
:+1:

File: auth_supplier/security/auth_supplier_security.xml (hunk 1 above)
Comment (hbrunn) [inline auth_supplier/security/auth_supplier_security.xml:7]: is generates a warning because the module doesn't depend on `portal`. Looking at the code, it seems it doesn't need to depend on portal, which would be the other fix. Or am I missing something?

Comment (pedrobaeza) [inline auth_supplier/security/auth_supplier_security.xml:7]: OK, thanks for the explanation

**Comment by pedrobaeza** [comment]
:+1:

**Comment by yajo** [comment]
:+1:
